clusters/howard-moomboo-space/clusters/test/limnocentral.yaml

```
@@ -52,10 +51,10 @@ spec:
   helmRelease:
     values: |-
       vcluster:
-        image: "rancher/k3s:v1.26.1-k3s1"
+        image: "rancher/k3s:v1.23.16-k3s1"
         extraArgs:
         - --kube-apiserver-arg --oidc-issuer-url=https://dex.howard.moomboo.space
-        - --kube-apiserver-arg --oidc-client-id=wg-kubelogin
+        - --kube-apiserver-arg --oidc-client-id=weave-gitops
```

```suggestion
        - --kube-apiserver-arg --oidc-client-id=wg-kubelogin
```

2539a29 (https://github.com/kingdonb/bootstrap-repo/commit/2539a29ff4107865a893412a251b29768c138bae)